src/web/assets/cp/src/js/EditableTable.js

```diff
@@ -97,29 +97,42 @@ Craft.EditableTable = Garnish.Base.extend(
 
       this.$addRowBtn = this.$table.next('.add');
       this.updateAddRowButton();
+
+      // If there's only one row, disable the action button
+      const $actionMenus = this.$tbody.find('.action-btn');
+      if (this.rowCount === 1) {
+        $actionMenus.attr('disabled', 'disabled').addClass('disabled');
+      }
+
       this.addListener(this.$addRowBtn, 'activate', 'addRow');
 
       if (this.settings.lazyInitRows) {
         // Lazily create the row objects
+        const interactionCallback = (ev) => {
+          const $target = $(ev.target);
+          const $tr = $target.closest('tr');
+          if ($tr.length && !$tr.data('editable-table-row')) {
+            const $textarea = $target.hasClass('editable-table-preview')
+              ? $target.next()
+              : null;
+            this.createRowObj($tr);
+            setTimeout(() => {
+              if ($textarea && !$textarea.is(':focus')) {
+                $textarea.focus();
+              }
+            }, 100);
+          }
+        };
+
         this.addListener(
           this.$tbody,
           'keypress,keyup,change,focus,blur,click,mousedown,mouseup',
-          (ev) => {
-            const $target = $(ev.target);
-            const $tr = $target.closest('tr');
-            if ($tr.length && !$tr.data('editable-table-row')) {
-              const $textarea = $target.hasClass('editable-table-preview')
-                ? $target.next()
-                : null;
-              this.createRowObj($tr);
-              setTimeout(() => {
-                if ($textarea && !$textarea.is(':focus')) {
-                  $textarea.focus();
-                }
-              }, 100);
-            }
-          }
+          interactionCallback
         );
+
+        if ($actionMenus.length) {
+          this.addListener($actionMenus, 'keypress,click', interactionCallback);
```

Comment:
@brandonkelly I had to add these listeners since the trigger activation doesn’t bubble up to the table. However, when I tried using the `activate` event, I was having issues. It was only getting triggered by mouse clicks but not keyboard activation. Wondering if you have any thoughts about a better way to go about this. 🤔